packages/analytics/CapacitorFirebaseAnalytics.podspec

```diff
@@ -23,6 +23,7 @@ Pod::Spec.new do |s|
 
   s.subspec 'Analytics' do |analytics|
     analytics.dependency 'FirebaseAnalytics', '~> 11.7.0'
+    analytics.dependency 'GoogleAppMeasurementOnDeviceConversion', '~> 11.7.0'
```

Comment:
Why not use `FirebaseAnalyticsOnDeviceConversion` as the documentation suggests? https://firebase.google.com/docs/tutorials/ads-ios-on-device-measurement/step-2#using_cocoapods